exporter/elasticsearchexporter/elasticsearch_bulk.go

```diff
@@ -103,7 +103,7 @@ func newElasticsearchClient(logger *zap.Logger, config *Config) (*esClientCurren
 		Header:    headers,
 
 		// configure retry behavior
-		RetryOnStatus:        retryOnStatus,
+		RetryOnStatus:        config.Retry.RetryOnStatus,
 		DisableRetry:         retryDisabled,
 		EnableRetryOnTimeout: config.Retry.Enabled,
```

Comment:
On a seperate note, should `EnableRetryOnTimeout` be equal to `!retryDisabled`?

Reply:
I checked the v7 estransport and EnableRetryOnTimeout usage is wrapped inside MaxRetries. Therefore, in practice, changing it to `!retryDisabled` will have no effect. But I agree that it will be clearer to do so. Let me make the change.

Reply:
hmm, I've changed my mind. The default in go-elasticsearch for DisableRetry and EnableRetryOnTimeout are both false, which makes sense. As a user I would not expect the client to retry the bulk request on timeout. Let me separate this discussion into an issue, whether the existing setting to enable retry on timeout to `config.Retry.Enabled` is desirable.